src/envoy/mixer/README.md

```diff
@@ -111,3 +119,24 @@ Mixer client can be configured to make Quota call for all requests.  If "quota_n
 ## How to pass some attributes from client proxy to mixer.
 
 Usually client proxy is not configured to call mixer (it can be enabled in the route opaque_config). Client proxy can pass some attributes to mixer by using "forward_attributes" field.  Its attributes will be sent to the upstream proxy (the server proxy). If the server proxy is calling mixer, these attributes will be sent to the mixer.
+
+
+## How to enable cache for Check calls
+
+Check calls can be cached. By default, it is not enabled. It can be enabled by supplying non-empty "check_cache_keys" string list in the mixer filter config. Only these attributes in the Check request, their keys and values, are used to calculate the key for the cache lookup. If it is a cache hit, the cached response will be used.  The cached response will be expired in 5 minutes by default. It can be overrided by supplying "check_cache_expiration_in_seconds" in the mixer filter config.
```

Comment:
I think it's worth mentioning that we intend to make this automatic such that it doesn't need to be configured by the operator.